openwisp_controller/geo/utils.py

```diff
@@ -18,4 +18,26 @@ def get_geo_urls(geo_views):
             geo_views.location_device_list,
             name='location_device_list',
         ),
+        path('api/v1/geo/floorplan/', geo_views.list_floorplan, name='list_floorplan'),
+        path(
+            'api/v1/geo/floorplan/<str:pk>/',
+            geo_views.detail_floorplan,
+            name='detail_floorplan',
+        ),
+        path('api/v1/geo/location/', geo_views.list_location, name='list_location'),
```

Comment:
| SN | api/v1/geo/location/                                 | Status |                                   Details                                   |
|:--:|------------------------------------------------------|:------:|:---------------------------------------------------------------------------:|
|  1 | [SU][O1A][O1O] Can view only appropriate locations   |  Pass  |                                                                             |
|  2 | [SU][O1A][O1O] Can create only appropriate locations |  Pass  |                                                                             |
|  3 | [SU][O1A] All errors reported together               |  Fail  | If title and geometry are missing, geometry is reported when title is fixed |